newton/_src/viewer/viewer_gl.py

```
@@ -2325,18 +2378,188 @@ def _edit_color3(
+            (0.6, (0.698, 0.165, 0.388)),
+            (0.8, (0.944, 0.403, 0.121)),
+            (1.0, (0.988, 0.998, 0.645)),
+        )
+        lut = np.empty((256, 4), dtype=np.uint8)
+        for index, value in enumerate(np.linspace(0.0, 1.0, 256, dtype=np.float32)):
+            for stop_index in range(len(inferno_stops) - 1):
+                t0, c0 = inferno_stops[stop_index]
+                t1, c1 = inferno_stops[stop_index + 1]
+                if value <= t1:
+                    alpha = 0.0 if t1 <= t0 else (float(value) - t0) / (t1 - t0)
+                    rgb = [int(round(255.0 * ((1.0 - alpha) * c0[channel] + alpha * c1[channel]))) for channel in range(3)]
```

Ruff (0.15.10)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)